google/datalab/bigquery/_udf.py

```diff
@@ -96,14 +96,21 @@ def _build_udf(name, code, return_type, params, language, imports):
     params = ','.join(['%s %s' % named_param for named_param in params])
     imports = ','.join(['library="%s"' % i for i in imports])
 
-    udf = 'CREATE TEMPORARY FUNCTION {name} ({params})\n' +\
-          'RETURNS {return_type}\n' +\
-          'LANGUAGE {language}\n' +\
-          'AS """\n' +\
-          '{code}\n' +\
-          '"""\n' +\
-          'OPTIONS (\n' +\
-          '{imports}\n' +\
-          ');'
+    if language.lower() == 'sql':
+        udf = 'CREATE TEMPORARY FUNCTION {name} ({params})\n' + \
+              'RETURNS {return_type}\n' + \
+              'AS (\n' + \
+              '{code}\n' + \
+              ');'
+    else:
+        udf = 'CREATE TEMPORARY FUNCTION {name} ({params})\n' +\
+              'RETURNS {return_type}\n' +\
+              ('LANGUAGE {language}\n' if language.lower() != 'sql' else '') +\
```

Comment:
Presumably the language is always != 'sql' at this point, and can be simplified. I doubt the generated LANGUAGE \n string is going to work in fact.

Reply:
Right you are, have added a fix